src/pyj/read_book/cfi.pyj

```diff
@@ -634,7 +619,6 @@ def scroll_to(cfi, callback, doc):  # {{{
         span.setAttribute('style', 'border-width: 0; padding: 0; margin: 0')
         r.surroundContents(span)
         scroll_viewport.scroll_into_view(span)
-        scroll_viewport.reset_transforms()  # needed for viewport_to_document()
```

Comment:
This is only called in preparation for viewport_to_document(), but that function is now part of ScrollViewport. So, just call it in ScrollViewport::viewport_to_document, and don't call it outside.